tortoise_interpreter.py

```diff
@@ -21,49 +21,49 @@ def sanitize_program(program):
 
 def navigate_command_functions(COMMANDS, program):
     command_to_parse = []
-    pen_state = 'UP'
+    draw = False
```

Comment:
So, this used to be `pen_state`, with a string value. The advice was to change the string to an object, so that we could store more information and name it well.

Now, we have `draw` which stores a boolean. This stores **_less_** information, and isn't as well named. We should have a talk about this one, because I think there's some miscommunication going on.